File: Examples/CSharp/WorkingWithResources/CreateResources.cs
/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Tasks for .NET API reference 
when the project is build. Please check https:// Docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Tasks for .NET API from https://www.nuget.org/packages/Aspose.Tasks/, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using https://forum.aspose.com/c/tasks
*/

namespace Aspose.Tasks.Examples.CSharp.WorkingWithResources
{
    using Aspose.Tasks.Saving;

    internal class CreateResources
    {
        public static void Run()
        {
            // The path to the documents directory.
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName);

            //ExStart:CreateResources
            //ExFor: Project.Resources
            //ExSummary: Shows how to create project resources.
            // Create project instance
            var project = new Project();

            // Add resources
            project.Resources.Add("Rsc");

            // Save project as XML
            project.Save(dataDir + "CreateResources_out.xml", SaveFileFormat.XML);
            //ExEnd:CreateResources
        }
    }
}

File: Examples/VisualBasic/WorkingWithResources/CreateResources.vb
Imports Aspose.Tasks.Saving

'
'This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Tasks for .NET API reference 
'when the project is build. Please check https://docs.nuget.org/consume/nuget-faq for more information. 
'If you do not wish to use NuGet, you can manually download Aspose.Tasks for .NET API from http://www.aspose.com/downloads, 
'install it and then add its reference to this project. For any issues, questions or suggestions 
'please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
'

Namespace WorkingWithResources
    Public Class CreateResources
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)

            ' Create project instance
            Dim project As New Project()

            ' ExStart:CreateResources
            ' Add resources
            Dim rsc1 As Resource = project.Resources.Add("Rsc")
            ' ExEnd:CreateResources

            ' Save project
            project.Save(dataDir & "CreateResources_out.xml", SaveFileFormat.XML)
        End Sub
    End Class
End Namespace
